### PR TITLE
Add 'cwd' REPL command to show the current working directory

### PR DESCRIPTION
**Pull request by ska80**

### CODE REVIEW

File: src/Idris/REPL.idr [hunk 1]
```diff
@@ -464,6 +464,7 @@ data REPLResult : Type where
   ErrorsBuildingFile : String -> List Error -> REPLResult
   NoFileLoaded : REPLResult
   ChangedDirectory : String -> REPLResult
+  WorkingDirectory : String -> REPLResult
```
Comment (abailly) [inline src/Idris/REPL.idr]: Maybe we could use a single `CurrentDirectory` result constructor for both commands?

**Review by abailly — COMMENTED** [review]
Minor simplification suggested

File: src/Idris/REPL.idr (hunk 1 above)
Comment (ska80) [reply src/Idris/REPL.idr]: Yes, I think it makes more sense.  I will do that.

File: src/Core/Context.idr [hunk 2]
```diff
@@ -1746,6 +1746,13 @@ setWorkingDir dir
          cdir <- coreLift $ currentDir
          put Ctxt (record { options->dirs->working_dir = cdir } defs)
 
+export
+getWorkingDir : {auto c : Ref Ctxt Defs} -> Core String
+getWorkingDir
+    = do defs <- get Ctxt
+         cdir <- coreLift $ currentDir
+         pure cdir
```
Comment (clayrat) [inline src/Core/Context.idr]: Can we just simplify the whole thing to `coreLift currentDir`? Doesn't seem like we even need to touch the context here.

Comment (ska80) [reply src/Core/Context.idr]: > Can we just simplify the whole thing to `coreLift currentDir`? Doesn't seem like we even need to touch the context here.

Will do that :)

**Comment by edwinb** [comment]
Looks a good idea to me, thanks.

**Pull request merged**